src/main/surveyScreen.java

```diff
@@ -0,0 +1,336 @@
+package main;
+
+import java.awt.event.*;
+import java.awt.*;
+import javax.swing.*;
+class surveyScreen  {
+
+	// JFrame
+	public static JButton[] progressButtons = new JButton[2];
+	static JFrame f;
+	public static JComboBox q0answers;
+	public static JComboBox q1answers;
+	public static JComboBox q2answers;
+	public static JComboBox q3answers;
+	public static JComboBox q4answers;
+	public static JComboBox q5answers; 
+	
+	public static JLabel halt;
+	
+	public static int civilEngineer = 0;
+	public static int softwareEngineer = 0;
+	public static int aerospaceEngineer = 0;
+	public static int mechanicalEngineer = 0;
+	public static int electricalEngineer = 0;
+	
+	static JPanel survey;
+
+	public static void createSurvey() {

```

Comment:
I think this should probably be the constructor "surveyScreen()" rather than a static method

Reply:
so that it can be called when we create a new instance of this class (or we might be doing this in another way, I'm not 100% sure yet)